### PR TITLE
Sprite scaling reference pages

### PR DESCRIPTION
**Pull request by ganicke**
Sprite scaling reference pages:

- [x] Create reference pages for the sprite scaling props and methods.
- [x] Create reference pages for the sprite scaling extension methods.
- [x] A few fixes for the `fx` and `fy` pages too.

Closes https://github.com/microsoft/pxt-arcade/issues/4562

RE: https://github.com/microsoft/pxt-arcade/issues/4590

### CODE REVIEW

**Comment by abchatra** [comment]
This looks good overall. I am going to merge this and we can fix if there is any issues later.

**Review by eanders-ms — COMMENTED** [review]
Looks great @ganicke!

File: libs/game/docs/reference/sprites/sprite/scale.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+* a [number](/types/number) that is the current scale for the sprite.
+
+## Set
+
+Set the scale for the sprite.
+
+```block
+let mySprite: Sprite = null
+
+mySprite.scale = 0
```
[the remark this answers is not in the document]
Comment (ganicke) [reply libs/game/docs/reference/sprites/sprite/scale.md:30]: Ah, yes good idea. I templated that from the other property pages. I'll use you suggestion. Thanks.

File: libs/game/docs/reference/sprites/sprite/sx.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+* a [number](/types/number) that is the current scale for the width of the sprite.
+
+## Set
+
+Set the width scale for the sprite.
+
+```block
+let mySprite: Sprite = null
+
+mySprite.sx = 0
```
[the remark this answers is not in the document]
Comment (ganicke) [reply libs/game/docs/reference/sprites/sprite/sx.md:30]: Right.

[the remark this answers is not in the document]
Comment (ganicke) [reply libs/game/docs/reference/sprites/sprite/sx.md:30]: Got that change put in https://github.com/microsoft/pxt-common-packages/pull/1322/commits/4f8a63a4452b007a43d94bb6f7a7dc7c39b4b281.

**Pull request merged**